OmgvaPOS/src/BusinessManagement/Services/BusinessService.cs

```diff
@@ -11,13 +12,15 @@ public class BusinessService(IBusinessRepository businessRepository) : IBusiness
         public List<BusinessDTO> GetBusinesses() {
             return _businessRepository.GetBusinesses().Select(b => b.ToBusinessDTO()).ToList();
         }
-        public BusinessDTO GetBusiness(long id) {
+        public BusinessDTO? GetBusiness(long id) {
             return _businessRepository.GetBusiness(id).ToBusinessDTO();
         }
         public BusinessDTO CreateBusiness(CreateBusinessRequest request) {
             return _businessRepository.CreateBusiness(request.ToBusiness()).ToBusinessDTO();
         }
-        public bool UpdateBusiness(BusinessDTO business) {
+        public bool UpdateBusiness(BusinessDTO business)
+        {
+            BusinessValidator.ValidateBusinessDTO(business);
             Business businessToUpdate = _businessRepository.GetBusiness((long)business.Id); //TODO: Unlikely but potential error here 
```

Comment:
If these exceptions handlings are fixed, then all the comments talking about errors should be deleted.